File: vm_Assignments/vmc/L2.6/LeapYear.c
#include<stdio.h>
void is_leap_year(int year)
{
    if(year % 400 ==0 || (year % 4 == 0 && year % 100 !=0))
    {
        printf("true\n");
    }
    else
    {
    printf("false\n");
    }
}

int main()
{
    is_leap_year(1792);
    is_leap_year(1796);
    is_leap_year(1800);
    is_leap_year(1804);
    is_leap_year(1900);
    is_leap_year(1904);
    is_leap_year(2000);
}

File: vm_Assignments/vm/Beginning/L2.6/LeapYear.py
def is_leap_year(year):
    if year % 400 == 0 or (year % 4 == 0 and year % 100 != 0):
        return True 
    else:
        return False

print(is_leap_year(1792))
print(is_leap_year(1796))
print(is_leap_year(1800))
print(is_leap_year(1804))
print(is_leap_year(1900))
print(is_leap_year(1904))
print(is_leap_year(2000))
print(is_leap_year(2000))
